scripts/reports/repo-health.sh

```diff
@@ -31,6 +31,16 @@ yum install gcc python3-devel git-lfs pango-devel -y
 pip3 install seaborn matplotlib pandas xlrd requests python-dateutil pyGithub jupyterlab django-weasyprint weasyprint==52.5
 git lfs install
 
+#Install alex
+yum install epel-release -y; curl https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+source ~/.bashrc
+nvm --version
+nvm install 14.20.0
+npm install alex --global -y
+alex --version
+alex_version=$(alex -v 2>&1); echo $alex_version | grep -o '[0-9.]*'
+node_version=$(node -v 2>&1); echo $node_version | grep -o '[0-9.]*'
```

Comment:
please add string `Node Version: ` in echo as well while printing version

Reply:
Done